go/README.md

```diff
@@ -1 +1,177 @@
+(You can look at other projects for an universal working version)
+
+## Package name
+
+A package name represents the content it houses. Inside there will be 1 or more files with functions that
+don't have the package name in front of it. Otherwise, when using a function, you get a stutter-effect,
+like this: database.DatabaseOpener(), while database.Opener() is enough.
+
+You can simplify a package name when it's a common abbreviation. database could be db, so db.Opener() can be used.
+
+When importing an external library (from aan zee), make it clear by using an alias, like:
+capErrors "bitbucket.org/aanzeeonline/capila/errors"
+libConv "bitbucket.org/aanzeeonline/libgoaz/conv"
+libCrypt "bitbucket.org/aanzeeonline/libgoaz/crypt"
+
+Don't use a name that it meaningless, like: util, common, misc.
+There is one exception to this: helpers.
+We will use helpers to store various helper-functions and over time, we can decide to group them into
+a package that is more suitable.
+
+## Imports
+
+order of imports
+
+- packages of current module
+- packages of aan zee lib's
+- the rest
+
+## Variable declaration
+
+Top level order of declaration:
+- constants
+- types
+- variables
+
+Global / package / function
+
+Don't use many Global var's.
+
+If a package var is used in more than 1 file, put it in a base.go
+
+Declare function var's mostly at the top of the function, or close to the first usage.
+
+## Functions and methods
+
+You can house more than 1 function inside a file (inside a package).
+The file name represents the content it houses.
+
+- naming
+	- give the function a name it represent.
+	- You can simplify a function name when it's a common abbreviation.
+		db.Initialize() could be db.Init()
+- no getters (the fact it begins with a capital and returns value, makes it a getter)
+- setters can be used
+- if crud, follow crud order inside source? (thoughts about this?)
+- global vs local
+  - first all global functions and methods (they start with capitals)
+  - local function below that (they start with lowercase)
+- return var's:
+  - when returning an error, it is always the last value that returns
+
+## Testing
+
+- code coverage (100% of relevant)
+- no . imports
+
+- Handlers should only test the basic working.
+- Services should test every scenario possible and should be tested with 100% codecoverage.
+  - if there are model validations, these should be tested too.
+- Repositories should test every scenario possible and should be tested with 100% codecoverage.
+- 
+
+## Comments
+
+- package
+  - doc
+    - Describes why the package exists
+    - it has the package format:
+      - Package the_name the_rest_of_the_text.
+- section
+  - special format:
+	/* ------------------------------- Imports --------------------------- */
+	/* ------------------------------- Constants/Types/Variables --------------------------- */
+	/* ------------------------------- Methods/Functions --------------------------- */
```

Comment:
Don't agree, if the order is already set by the code convention the specific sections don't have to be named.